src/niscope/custom_types/measurement_stats.py

```diff
@@ -0,0 +1,43 @@
+class MeasurementStats(object):
```

Comment:
do you have unit tests for this small class?

Reply:
No, only system tests. I'll make sure to write some unit tests for it.

Reply:
So let's do this: Make a separate PR for adding the class with its accompanying unit tests. Is that ok?

Reply:
Sounds fine to me. I'll break it off into a small PR of its own shortly.

Reply:
Tests already exist, and all changes necessary for adding MeasurementStats are already in this PR. Will keep it as one PR, since MeasurementStats is intended to accompany fetch_measurement_stats.